source/common/router/router_ratelimit.cc

```diff
@@ -263,12 +264,80 @@ QueryParameterValueMatchAction::buildQueryParameterMatcherVector(
   return ret;
 }
 
+absl::Status resolveHitsAddendSource(const envoy::config::route::v3::RateLimit::HitsAddend& config,
```

Comment:
this and getHitsAddendViaProvider are simply moved from the ratelimit_config.cc below, and their logic hasn't changed